packages/engine-ui/src/types.ts

```diff
@@ -23,3 +23,33 @@ export type Vehicle = {
 }
 
 export type NotificationType = Vehicle | Booking
+
+export interface PlanVehicle {
```

Comment:
What is PlanVehicle? Is there anything better to name this? :)

Reply:
It is...vehicle in plan. I didn't create this in this PR, only moved it. But we should go over types and naming in this project

Reply:
Okay. Makes me think of something that is both a vehicle and a plan at the same time.. dogduck.